player/player.cs

```diff
@@ -0,0 +1,21 @@
+using Godot;
+using System;
+
+public class player : KinematicBody2D
```

Comment:
Is this file needed?  I think only `Player.cs` is used.

Reply:
I removed the file